my-sms/app/controllers/students_controller.rb

```diff
@@ -9,7 +9,9 @@ class StudentsController < ApplicationController
   # GET /students
   # GET /students.json
   def index
-    @students = Student.all
+    @students = StudentDecorator.decorate_collection(
+      Student.paginate(page: params[:page], per_page: 10)
```

Comment:
As mentioned in Matt's PR, consider moving this magic number (`10`) to a constant or appropriately named variable (eg. `DEFAULT_PER_PAGE`)

Reply:
Fixed in 39a3030580d87e5999959ca7d9f866b58f4e14e3